tests/sanity.test.ts

```diff
@@ -0,0 +1,15 @@
+import { AxiosDispatcher } from 'HTTP/AxiosDispatcher';
```

Comment:
I think that there are many things to be tested here. Or is it not a part of the task?

Reply:
I had many basic tests in scalecube, when I was working on Transport and its providers. Suggest to wait Idans confirmation that this amount of tests is fine, cause I feel it's not.